### PR TITLE
Update psammead dependencies

### LINKED ISSUE #1172: Bump Psammead-caption and other pending packages
~~Blocked on https://github.com/bbc/psammead/issues/270~~

Required for integrating https://github.com/bbc/psammead/issues/271

**Is your feature request related to a problem? Please describe.**
Due to new UX requirements we need to include new fonts in simorgh

**Describe the solution you'd like**
~Include `Reith Sans Bold`, `Reith Sans Italic` and `Reith Sans Italic Bold` here https://github.com/bbc/simorgh/blob/latest/webpack.config.client.js#L82-L99~ 
This has been done, but we should also pull in any changed psammead packages. This can be done with
```
npx npm-check-updates '/@bbc\//' -u -a
```

 Please compare the packages pulled in, and referencing their changelog list the expected changes for a tester to look over. 

**Testing notes**
[Tester to complete]

Dev insight: Visual regression testing required.

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves #1172

**Overall change:** Pulls in latest Psammead dependencies. Needed for latest caption styling.

**Code changes:**

- Updated all latest psammead dependencies except `gel-foundations` (current version is buggy on Psammead)
- Resolved `js-yaml` subdependencies to be a version that doesn't have a vulnerability
- Updates snapshots (Psammead components have updated to use `rem` over `em`)

Changes from change logs:

```
"@bbc/psammead-brand": Uses rem for media queries.
"@bbc/psammead-caption": Remove background colour, update colour to Cloud Dark. Use font style italic. Also ensures left/right padding remains tight in context of article.
"@bbc/psammead-copyright": Updates dependencies
"@bbc/psammead-image": n/a (Update storybook badge url)
"@bbc/psammead-image-placeholder": n/a (Update storybook badge url)
"@bbc/psammead-inline-link": n/a (Update storybook badge url)
"@bbc/psammead-paragraph": increases padding-bottom as per designs
"@bbc/psammead-sitewide-links": uses rem for media queries.
"@bbc/psammead-visually-hidden-text": n/a (Update storybook badge url)
```

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,20 +78,20 @@
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.3.4",
-    "@bbc/gel-foundations": "^0.2.4",
+    "@bbc/gel-foundations": "^0.3.0",
```
Comment (dr3) [inline package.json]: Last time I did this James wanted a list of all possible changes to these packages like https://github.com/bbc/simorgh/pull/1394. Its made by comparing the versions and manually copying the changes in their changelog

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — DISMISSED** [review]

**Review by dr3 — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]
👍 Looks good. I noted that now that we have removed the background colour of the caption, the padding (1rem, 16px) makes the text look indented. 

Screen Shot of caption with 1rem padding

We'll likely want to check with UX whether this is desired.

**Comment by ChrisBAshton** [comment]
Failed UX review, raised bug with gel-foundations: https://github.com/bbc/psammead/issues/388

**Comment by ChrisBAshton** [comment]
> 👍 Looks good. I noted that now that we have removed the background colour of the caption, the padding (1rem, 16px) makes the text look indented.
> 
> Screen Shot of caption with 1rem padding
> 
> We'll likely want to check with UX whether this is desired.

We've now raised https://github.com/bbc/psammead/issues/382 for this.

**Comment by ChrisBAshton** [comment]
Given we've still work to do on figcaption, let's fix https://github.com/bbc/psammead/issues/382 before continuing with this PR.

**Comment by ChrisBAshton** [comment]
Blocked on https://github.com/bbc/psammead/pull/393.

**Comment by ChrisBAshton** [comment]
Now blocked on #398.

**Comment by ChrisBAshton** [comment]
Now blocked on https://github.com/bbc/psammead/pull/381.

**Comment by ChrisBAshton** [comment]
Now blocked on https://github.com/bbc/psammead/pull/406.

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

**Review by dr3 — DISMISSED** [review]

**Comment by ChrisBAshton** [comment]
Blocked on https://github.com/bbc/psammead/pull/410 :(

File: .env [hunk 2]
```diff
@@ -1,4 +1,7 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```
Comment (sareh) [inline .env]: Can you please revert this file's changes?

**Review by sareh — APPROVED** [review]
👍 Looks great.

**Review by pjlee11 — APPROVED** [review]

**Comment by dr3** [comment]
Have popped this straight into ready for test, This can be UX reviewed in parallel. Any changes will likely be raised in new issues

**Pull request merged**